os/include/tinyara/fs/ioctl.h

```diff
@@ -93,6 +93,7 @@
 #define _RTCBASE        (0x1800)	/* RTC ioctl commands */
 #define _FOTABASE       (0x1900)	/* FOTA ioctl commands */
 #define _GPIOBASE       (0x2000)	/* GPIO ioctl commands */
+#define _TESTFWIOCBASE (0xfe00)	/* KERNEL TEST FW module ioctl commands */
```

Comment:
1. Please check alignments to value and comment
2. Not 0x2100? Why do you set 0xfe00?